### PR TITLE
Fixed #27156 - Added a test for GET for setting HTTPRequest.encoding.

### PR DESCRIPTION
**Pull request by PREM1980**
Part of ticket 27113 we included a test for POST for setting HTTPRequest.encoding. We decided to implement the GET test in the later ticket and this ticket was opened for it.

### CODE REVIEW

**Comment by timgraham** [comment]
This isn't correct. The new test should use the same pattern as the POST test (check request.GET, change encoding, check for the new request.GET). As noted on the ticket, the behavior is currently broken, thus if the test is passing without any changes, it isn't correct.

File: tests/requests/tests.py [hunk 1]
```diff
@@ -554,7 +554,7 @@ def read(self, len=0):
         with self.assertRaises(UnreadablePostError):
             request.body
 
-    def test_set_encoding_clears_POST(self):
+    def test_set_encoding_clears(self):
```
Comment (timgraham) [inline tests/requests/tests.py]: Add a separate `test_set_encoding_clears_GET` method.

**Comment by timgraham** [comment]
Please send a new PR if you correct it.

**Comment by PREM1980** [comment]
Sure, will submit a new request.

**Pull request closed without merging**